pom.xml

```diff
@@ -72,28 +69,39 @@
 			<artifactId>aws-cloudwatch-library</artifactId>
 			<version>1.11.136</version>
 			<optional>true</optional>
+			<exclusions>
+				<!-- Provided by apache-httpcomponents-client-4-api plugin -->
+				<exclusion>
+					<groupId>org.apache.httpcomponents</groupId>
+					<artifactId>httpclient</artifactId>
+				</exclusion>
+				<exclusion>
+					<groupId>org.apache.httpcomponents</groupId>
+					<artifactId>httpcore</artifactId>
+				</exclusion>
+			</exclusions>
+		</dependency>
+		<dependency>
+			<groupId>org.jenkins-ci.plugins</groupId>
+			<artifactId>apache-httpcomponents-client-4-api</artifactId>
+			<version>4.5.14-150.v7a_b_9d17134a_5</version>
+			<optional>true</optional>
+		</dependency>
+		<dependency>
+			<groupId>org.jenkins-ci.plugins</groupId>
+			<artifactId>bouncycastle-api</artifactId>
+			<version>2.28</version>
+			<optional>true</optional>
```

Comment:
why adding this dependency ?

Reply:
=> commit e299c74fdef19af0a0cc3f334fffd8a85349c004